megamek/src/megamek/common/Entity.java

```diff
@@ -249,6 +249,7 @@ public abstract class Entity extends TurnOrdered implements Transporter, Targeta
 
     protected int facing = 0;
     protected int sec_facing = 0;
+    protected GamePhase twistedPhase = null;
```

Comment:
We probably want to make sure this is cleared out when resetting an entity as well.

Reply:
`twistedPhase` gets reset at the start of every Round, when `newRound()` is called:

```
        setAlreadyTwisted(false);
```
We don't want to reset it every phase as it has to persist between Targeting and Firing phases.